patches/build-rust-cargo_crate.gni.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/build/rust/cargo_crate.gni b/build/rust/cargo_crate.gni
+index 9368a9ca2086b4e1b1c172c4ece4aa6715d4c798..2cc3ec784a8630512bf78c74b5b12fee051a4a35 100644
+--- a/build/rust/cargo_crate.gni
++++ b/build/rust/cargo_crate.gni
+@@ -190,7 +190,7 @@ template("cargo_crate") {
+                                  "rustenv",
+                                  "dev_deps",
+                                ])
+-    forward_variables_from(invoker, TESTONLY_AND_VISIBILITY)
++    forward_variables_from(invoker, [ "testonly" ])
```

Comment:
a lot of the upstream crates in third_party/rust have limited visibility and patching them all would require a lot of patches. I spoke to chromium rust devs and I'm going to submit a patch to add a gn arg for the default visibility so we can override it